From ff8456dd3b42215e024c04de812ecca33d17f6d6 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 14 May 2009 20:47:10 -0700
Subject: Staging: cpc-usb: add driver to the build

This adds the cpc-usb driver to the kernel build

Cc: Sebastian Haas <haas@ems-wuensche.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/Kconfig          |    2 ++
 drivers/staging/Makefile         |    1 +
 drivers/staging/cpc-usb/Kconfig  |    4 ++++
 drivers/staging/cpc-usb/Makefile |    3 +++
 4 files changed, 10 insertions(+)
 create mode 100644 drivers/staging/cpc-usb/Kconfig
 create mode 100644 drivers/staging/cpc-usb/Makefile

--- /dev/null
+++ b/drivers/staging/cpc-usb/Kconfig
@@ -0,0 +1,4 @@
+config USB_CPC
+	tristate "CPC CAN USB driver"
+	depends on USB
+	default n
--- /dev/null
+++ b/drivers/staging/cpc-usb/Makefile
@@ -0,0 +1,3 @@
+obj-$(CONFIG_USB_CPC)	+= cpc-usb.o
+
+cpc-usb-y := cpc-usb_drv.o sja2m16c_2.o
--- a/drivers/staging/Kconfig
+++ b/drivers/staging/Kconfig
@@ -121,5 +121,7 @@ source "drivers/staging/octeon/Kconfig"
 
 source "drivers/staging/vt6655/Kconfig"
 
+source "drivers/staging/cpc-usb/Kconfig"
+
 endif # !STAGING_EXCLUDE_BUILD
 endif # STAGING
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -42,3 +42,4 @@ obj-$(CONFIG_LINE6_USB)		+= line6/
 obj-$(CONFIG_USB_SERIAL_QUATECH_ESU100)	+= serqt_usb/
 obj-$(CONFIG_OCTEON_ETHERNET)	+= octeon/
 obj-$(CONFIG_VT6655)		+= vt6655/
+obj-$(CONFIG_USB_CPC)		+= cpc-usb/
